omnipaxos/src/messages.rs

```diff
@@ -73,6 +73,8 @@ pub mod sequence_paxos {
         pub sync_idx: u64,
         /// The decided index
         pub decided_idx: u64,
+        /// The accepted index
+        pub accepted_idx: u64,
```

Comment:
Why is this needed? Isn't it sync_idx + suffix.len() + 0 or 1 (if StopSign is some)? I.e., we can calculate this by calling `self.internal_storage.get_accepted_idx()` at the receiver.

It is probably equally correct to explicitly send the accepted_idx. But I prefer if we use the `self.internal_storage.get_accepted_idx()` for consistency (and avoid any potential bugs)